commands/dev-deploy.js

```diff
@@ -62,5 +65,7 @@ async function createDevAccountIfNeeded({ near, keyStore, networkId, init }) {
     await near.accountCreator.createAccount(accountId, keyPair.publicKey);
     await keyStore.setKey(networkId, accountId, keyPair);
     await writeFile(accountFilePath, accountId);
```

Comment:
let's add TODO / issue to remove old file once examples / create-near-app are updated